File: 2356-largest-combination-with-bitwise-and-greater-than-zero/2356-largest-combination-with-bitwise-and-greater-than-zero.java
class Solution {
    public int largestCombination(int[] candidates) {
        int maxCount=0,count;
        //System.out.println(setBit);
        for(int i=0;i<24;i++ ){
            count=0;
            for(int num:candidates){
                if((num&(1<<i))!=0){
                    count++;
                }
                maxCount=Integer.max(count,maxCount);
            }
        }
        return maxCount;
    }
}

File: 2356-largest-combination-with-bitwise-and-greater-than-zero/2356-largest-combination-with-bitwise-and-greater-than-zero.py
class Solution:
    def largestCombination(self, candidates: List[int]) -> int:
        maxCount=0
        for i in range(24):
            count=0
            for num in candidates:
                if (num & 1<<i)!=0:
                    count+=1
            maxCount=max(count,maxCount)
        return maxCount